zip-1014.rst

```diff
@@ -337,13 +348,13 @@ Members of ZF's Board of Directors MUST NOT hold equity in ECC or have current
 business or employment relationships with ECC, except as provided for by the
 grace period described below.
 
-Grace period: members of the board who hold ECC equity (but do not have other
-current relationships to ECC) may dispose of their equity, or quit the Board,
-by 1 November 2021. (The grace period is to allow for orderly replacement, and
-also to allow time for ECC corporate reorganization related to Dev Fund
-receipt, which may affect how disposition of equity would be executed.)
+Grace period: members of the ZF board who hold ECC equity
+(but do not have other current relationships to ECC) may dispose of their equity,
```

Comment:
What does "current relationships" mean here?

Is it meant to say "MUST dispose of their equity or quit the board, by 1 November 2021."?

Reply:
I'm not touching this language for now, i reverted the unintentional spacing changes. I think it said "may dispose" in the grace period paragraph just because it weakens the "MUST NOT hold equity in ECC" above, it's not imposing any additional MUST.